l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -640,7 +640,8 @@ def setDettaglioLinee(self, invoice, body):
             if line.product_id:
                 if line.product_id.default_code:
                     CodiceArticolo = CodiceArticoloType(
-                        CodiceTipo='ODOO',
+                        CodiceTipo=self.env['ir.config_parameter'].sudo(
+                            ).get_param('fatturapa.codicetipo.odoo', 'ODOO'),
```

Comment:
Stavo pensando: in ambiente multi azienda potrebbe servire avere un `CodiceTipo` diverso per azienda?

Reply:
Ho provato su runbot ad impostare il parametro di sistema `fatturapa.codicetipo.odoo` e generare l'XML, ma l'XML contiene sempre `<CodiceTipo>ODOO</CodiceTipo>`.